evm/deploy-contract.js

```diff
@@ -352,6 +353,9 @@ if (require.main === module) {
         new Option('-m, --deployMethod <deployMethod>', 'deployment method').choices(['create', 'create2', 'create3']).default('create2'),
     );
     program.addOption(new Option('--ignoreError', 'ignore errors during deployment for a given chain'));
+    program.addOption(new Option('--signers <signers>', 'signer addresses for multisig').env('SIGNERS'));
+    program.addOption(new Option('--threshold <threshold>', 'multisig threshold').env('THRESHOLD'));
+    program.addOption(new Option('--minTimeDelay <minTimeDelay>', 'minimum time delay').env('MIN_DELAY'));
```

Comment:
since it's not scalable to have separate flag for every arg, this should be passed generically. Specifically, the flag can just be `--args`, and if provided, it can override the `contractConfig` values. E.g.
`--args '{"signers": ["0x1...", "0x2.."], "threshold": 3}'`
This can be parsed as JSON, and then used by the script and also update the config object so it gets saved locally.